executor/show_test.go

```diff
@@ -296,6 +296,26 @@ func (s *testSuite) TestShow(c *C) {
 			"  `e` bigint(20) DEFAULT NULL\n" +
 			") ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin",
 	}
+	for i, r := range row {
+		c.Check(r, Equals, expectedRow[i])
+	}
+
+	// The following two cases are for https://github.com/pingcap/docs-cn/issues/537
```

Comment:
We better not reference a issue in `docs-cn`.

Reply:
Agree

Reply:
Done.